server/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -1137,6 +1137,31 @@ public void test_c_options_in_properties() throws Exception {
         }
     }
 
+    @Test
+    public void test_insert_array_works_in_simple_query_mode() throws Exception {
```

Comment:
```suggestion
    public void test_insert_array_in_simple_query_mode() throws Exception {
```

No strong opinion and just a suggestion, but `works` in a test case name feels redundant.